tests/test_infer/test_llama_infer.py

```diff
@@ -52,30 +50,45 @@ def init_to_get_rotary(self, base=10000):
 def run_llama_test(test_config):
 
     llama_model_path = "/data/scratch/llama-7b-hf"
+
     if os.path.isdir(llama_model_path) is False:
-        return
+        sub_model_zoo = model_zoo.get_sub_registry('transformers_llama_for_casual_lm')
```

Comment:
NOTE: The registered module has a typo in 'causal' in tests/kit/model_zoo/transformers/llama.py:
```python
model_zoo.register(name='transformers_llama_for_casual_lm',
```
Thus retrieving registry with `transformers_llama_for_causal_lm` will cause a not-found error. 

The typos shall be fixed in another NFC PR (registered name with typos are used in other files)